install_everything.sh

```diff
@@ -41,16 +41,16 @@ else
     echo "libbacktrace is already installed"
 fi
 
-#libpfm-4.10.1
-PFM_ROOT=$ROOT_DIR/libpfm-4.12.0
-PFM_INSTALL_ROOT=$ROOT_INSTALL_DIR/libpfm-4.12.0
+
+#libpfm
+PFM_ROOT=$ROOT_DIR/libpfm
+PFM_INSTALL_ROOT=$ROOT_INSTALL_DIR/libpfm
 
 if ! [ -d $PFM_INSTALL_ROOT ]; then
     echo "Installing libpfm..."
     rm -rf $PFM_ROOT
-    git clone https://git.code.sf.net/p/perfmon2/libpfm4 $PFM_ROOT || exit 1
+    git clone git://git.code.sf.net/u/jklinkenberg/perfmon2 $PFM_ROOT || exit 1
```

Comment:
Why change the perfmon repo ? Are there changes in your repo that are needed and can't be merged into the "official" perfmon repo ?

Reply:
Hi, I actually reverted that change in a later commit as it is not necessary any more.
The tab "Files changed" should not list install_everything.sh any more.

Background:
We have Icelake system where PEBS was not working for loads due to some missing code in the kernel source for that architecture. After reporting the issue, Intel wrote a kernel fix that now already be published.
My temporary perform2 repo implemented a workaround for Icelake as the new kernel was not yet available.

If it is cleaner I can also create a PR not including changes for that file at all